# Genome Research
- [Hsiao et al., 2016. Alternative splicing modulated by genetic variants demonstrates accelerated evolution regulated by highly conserved proteins.](http://genome.cshlp.org/content/early/2016/03/09/gr.193359.115.abstract)
    - Identification of genetic variants associated with alternative splicing.
    - Exons, introns, and SNPs associated with alternative splicing show accelerated evolution in primates.
    - Propose that the machinery is highly conserved, and shows strong allele bias.
- [Keith et al., 2015. High mutational rates of large-scale duplication and deletion in Daphnia pulex.](http://genome.cshlp.org/content/early/2015/12/14/gr.191338.115.abstract)
	- Mutation accumulation (MA) study in _Daphnia_
	- Estimates of base substitution rate similar to other Eukaryotes
	- Estimates of CNV much higher than those observed in other MA studies
	- Evidence of meiotic and mitotic gene conversion (GC), with mitotic tract lengths being longer
	- Evidence of purifying selection against CNV in natual populations

- [Li et al., 2015. Phylogenomic evidence for ancient hybridization in the genomes of living cats (Felidae)](http://genome.cshlp.org/content/early/2015/11/30/gr.186668.114.abstract)
	- Disconcordance between autosomal, mt-linked, X-linked, and Y-linked phylogenies

- [He et al., 2015. A probabilistic method for testing and estimating selection differences between populations. 25(12):1903-9](http://genome.cshlp.org/content/25/12/1903.abstract)
	- Probabilistic models of drift and selective coefficients to test for differential selection between populations
	- Interesting: difference in selective coefficients is approximately normally distributed
