# Molecular Biology and Evolution Articles
### The folowing is a list of papers from MBE that looks insteresting
1. [Nakagome, S., Alkorta-Aranburu, G., Amato, R., Howie, B., Peter, B. M., Hudson, R. R., &amp; Di Rienzo, A. (2015). Estimating the ages of selection signals from different epochs in human history. Molecular biology and evolution, msv256.doi: 10.1093/molbev/msv256](http://mbe.oxfordjournals.org/content/early/2015/11/05/molbev.msv256.short?rss=1)
   - The update version of ms, but need to ask from Richard Hudson
   - The method mentioned in this paper could be used in our Barley deleterious mutation project
2. [Chen, Hua, Jody Hey, and Kun Chen. (2015). Inferring Very Recent Population Growth Rate from Population-Scale Sequencing Data: Using a Large-Sample Coalescent Estimator. Molecular biology and evolution. 32 (11): 2996-3011. doi: 10.1093/molbev/msv15]
(http://mbe.oxfordjournals.org/content/32/11/2996.abstract)
   - A computationally efficient method for simultaneous estimation of population size, the rate, and onset time of population growth in the very recent history
   - Using the pattern of the total number of segregating sites as a function of sample size 
   - Flexible to model population history with multiple growth stages or other epochs
3. [Cheeseman, Ian H., et al. (2015). Population structure shapes copy number variation in malaria parasites. Molecular biology and evolution (2015): msv282.]
(http://mbe.oxfordjournals.org/content/early/2015/11/26/molbev.msv282.abstract)
   - Copy number variation and population structure
   - Maybe we can also examine copy number variation in Barley
4. [Castellano, D., Coronado-Zamora, M., Campos, J. L., Barbadilla, A., & Eyre-Walker, A. (2015). Adaptive Evolution Is Substantially Impeded by Hill–Robertson Interference in Drosophila. Molecular biology and evolution, msv236.] (http://mbe.oxfordjournals.org/content/33/2/442.full.pdf+html)
   - Quantify the overall impact of Hill–Robertson Interference on the rate of adaptive evolution for Drosophila genome
5. [Stevison, L. S., Woerner, A. E., Kidd, J. M., Kelley, J. L., Veeramah, K. R., McManus, K. F., ... & Wall, J. D. (2015). The time-scale of recombination rate evolution in great apes. Molecular biology and evolution, msv331.]
(http://mbe.oxfordjournals.org/content/33/4/928.full)
   - Analyze within- and between-species genome-wide recombination rate variation in several close relatives
6. [de Filippo, C., Key, F. M., Ghirotto, S., Benazzo, A., Meneu, J. R., Weihmann, A., ... & Andrés, A. M. (2016). Recent selection changes in human genes under long-term balancing selection. Molecular biology and evolution, msw023](http://mbe.oxfordjournals.org/content/33/6/1435.full.pdf+html)
   - loci show hallmark signatures of long-term balancing selection in African populations, but not in Eurasian populations
   - Balancing selection thus likely served as a source of functional alleles that mediated subsequent adaptations to novel environments
7. [Zhang, M., Zhou, L., Bawa, R., Suren, H., & Holliday, J. A. (2016). Recombination rate variation, hitchhiking, and demographic history shape deleterious load in poplar. Molecular Biology and Evolution, msw169](http://mbe.oxfordjournals.org/content/early/2016/08/10/molbev.msw169.abstract)
   - Species limits range shape the deleterious load
   - Closely related to our invertion and range limits project
8. [Lenz, T. L., Spirin, V., Jordan, D. M., & Sunyaev, S. R. (2016). Excess of deleterious mutations around HLA genes reveals evolutionary cost of balancing selection. Molecular Biology and Evolution](http://mbe.oxfordjournals.org/content/early/2016/06/29/molbev.msw127.full)
   - Test the hypothesis that balancing selection may counteract purifying selection in neighboring regions and thus maintain deleterious variants at higher frequency than expected from their detrimental fitness effect
   - Using whole-exome sequencing data from 6,500 human individuals
9. [Kono, T. J., Fu, F., Mohammadi, M., Hoffman, P. J., Liu, C., Stupar, R. M., ... & Morrell, P. L. (2016). The role of deleterious substitutions in crop genomes.Molecular Biology and Evolution](http://mbe.oxfordjournals.org/content/early/2016/06/23/molbev.msw102.full)
